src/transaction/log_page_buffer.c

```diff
@@ -6177,6 +6179,14 @@ logpb_remove_archive_logs (THREAD_ENTRY * thread_p, const char *info_reason)
 
   last_deleted_arv_num--;
 
+  vacuum_first_pageid = vacuum_min_log_pageid_to_keep (thread_p);
+  if (logpb_is_page_in_archive (vacuum_first_pageid))
```

Comment:
I think ```vacuum_first_pageid != NULL_PAGEID``` should be checked before this condition.

Reply:
You are right. I will fix it.